gaphor/diagram/shapes.py

```diff
@@ -26,8 +35,19 @@ def draw_default_box(box, context, bounding_box):
 
 
 class Box:
-    def __init__(self, *children, style={}, draw=draw_default_box):
+    """
+    A box like shape.
+
+    CSS properties:
+    - min-height
+    - min-width
+    - padding: a tuple (top, right, bottom, left)
```

Comment:
It may be more readable here if we use an [Enum](https://docs.python.org/3/library/enum.html). Then below we could reference the padding more explicitly like:
```
x = bounding_box.x + padding[Padding.LEFT]
```

Reply:
That's a good idea.